docs/build/program/apis/ml-training-client.md

```diff
@@ -121,6 +121,65 @@ For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/
+- `model_version` ([str](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str)) (required): The version of the ML model you're training. This string must be unique from any previous versions you've set.
+- `model_type` ([ModelType.ValueType](https://python.viam.dev/autoapi/viam/gen/app/mltraining/v1/ml_training_pb2/index.html#viam.gen.app.mltraining.v1.ml_training_pb2.ModelType)) (required): The type of the ML model. Options: `ModelType.MODEL_TYPE_UNSPECIFIED`, `ModelType.MODEL_TYPE_SINGLE_LABEL_CLASSIFICATION`, `ModelType.MODEL_TYPE_MULTI_LABEL_CLASSIFICATION`, `ModelType.MODEL_TYPE_OBJECT_DETECTION`.
+- `tags` ([List(str)](https://docs.python.org/3/library/stdtypes.html#typesseq-list)) (required): The labels to train on.
+
+**Returns:**
+
+- ([str](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str)): The ID of the training job created.
+
+```python {class="line-numbers linkable-line-numbers"}
+dataset_id = await ml_training_client.submit_training_job(
+  "3988fd7d-b1c0-4f36-9842-0c666f374d3c",
+  "66491a0666a6c8aa4a33e43e",
+  "your-model-name",
+  "1",
+  "ModelType.MODEL_TYPE_SINGLE_LABEL_CLASSIFICATION",
+  tags)
+```
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/app/ml_training_client/index.html#viam.app.ml_training_client.MLTrainingClient.submit_training_job).
+
+{{% /tab %}}
+{{% tab name="Flutter" %}}
+
+**Parameters:**
+
+- `datasetId` [(String)](https://api.flutter.dev/flutter/dart-core/String-class.html) (required): The ID of the dataset to train the ML model on. To retrieve this, navigate to your dataset's page in the [Viam app](https://app.viam.com/data/datasets), click **...** in the left-hand menu, and click **Copy dataset ID**.
+- `modelName` [(String)](https://api.flutter.dev/flutter/dart-core/String-class.html) (required): The name of the ML model to train.
+- `modelType` [(ModelType)](https://flutter.viam.dev/viam_protos.app.ml_training/ModelType-class.html) (required): The type of the ML model. Options: `MODEL_TYPE_UNSPECIFIED`, `MODEL_TYPE_SINGLE_LABEL_CLASSIFICATION`, `MODEL_TYPE_MULTI_LABEL_CLASSIFICATION`, or `MODEL_TYPE_OBJECT_DETECTION`.
+- `modelVersion` [(String)](https://api.flutter.dev/flutter/dart-core/String-class.html) (required): The version of the ML model you're training. This string must be unique from any previous versions you've set.
+- `organizationId` [(String)](https://api.flutter.dev/flutter/dart-core/String-class.html) (required): The ID of the organization to submit the training job to. To retrieve this, expand your organization's dropdown in the top right corner of the [Viam app](https://app.viam.com/), select **Settings**, and copy **Organization ID**.
+- `tags` [(List)](https://api.flutter.dev/flutter/dart-core/List-class.html)<[String](https://api.flutter.dev/flutter/dart-core/String-class.html)> (required): The labels to train on.
+
+**Returns:**
+
+- None.
```

Comment:
It seems like the Flutter SDK returns the SubmitTrainingJobResponse, which contains the job id https://flutter.viam.dev/viam_protos.app.ml_training/SubmitTrainingJobResponse-class.html

Reply:
@andf-viam I think this might have been a scraping issue